charts/data-dashboard-backend/values.yaml

```diff
@@ -0,0 +1,118 @@
+# Default values for data-dashboard-backend.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- Number of replicas to deploy
+replicaCount: 2
```

Comment:
If it's not necessary, I suggest the default number of replicas to be 1

Reply:
I will reduce the number of replicas to 1.